README.md

```diff
@@ -29,8 +29,6 @@ bundle install
 Then, set it up:
 
 ```
-> bin/rails generate rspec:install
-> bin/rails generate apitome:install
 > bin/rails generate stitches:api
```

Comment:
`rspec:install` and `apitome:install` are implicitly required to run before `stitches:api`, so rather than make the user run them I've added it to the `stitches:api` generator.